### DISCUSSION TITLE
Custom Node Hunting

### DISCUSSION THREAD

**Discussion by SammBrooks** [GimelStudio/GimelStudio#200] [Q&A]
Curious if there's anyone has created a place to share add-ons/nodes.

I love the concept of this program (I'm an absolute hog for node-based workflows), but I would like to get more functionality out of it. 
Since it's still very early in development and I'm a dunce when it comes to coding, I'm genuinely intrigued if someone out there has been able to whip-up some nifty stuff that could add some new functionality.

Cheers.

**Comment by Correct-Syntax** [comment]
Hello @SammBrooks, core developer of Gimel Studio here. Glad to hear you're liking the project. 👏 

What functionality are you looking for specifically? As Gimel Studio is still in early development I'm not sure how many in the community have experimented with custom nodes, but **anyone is welcome to reply to this thread with their custom node scripts**.

The custom nodes are a python script, sometimes with some GLSL. https://github.com/GimelStudio/GimelStudio/tree/master/src/nodes/customnodes would give you a good idea with some examples.

_As an aside, I'll just let you know that in this community we appreciate avoiding vulgar language of any kind, even if it is censored. Thanks for understanding._

**Reply by SammBrooks** [reply]
Nothing specific at the moment. I'm just interested to play around with it as much as I can as it gets developed out.
I currently use Photoshop for my image manipulation, and all of the alternatives don't really strike my fancy.
So I'm super excited to see how this all plays out once it's completed  and things like (presumably) masking are introduced, and all the bugs have been ironed out.

Until then though, just looking for some extra toys to play around with. So I'll most definitely keep keeping an eye out for people who have posted scripts that can be used (would tackle them myself, but I don't know anything about coding and don't have the time to learn), as well as keep an eye on the project at large.
 
Big hopes for the future of this one, it's right up my alley.

_Fixed my comment btw, and I'll avoid it in the future, sorry._

**Reply by Correct-Syntax** [reply]
Ok, was asking because its helpful to know precisely what interests people in the project. Software development isn't an overnight thing -especially when it comes to graphics software- so I its likely going to be awhile before it gets even close to normal use level functionality.

**Reply by SammBrooks** [reply]
Oh, of course.
All good things take time- especially good programs.

If you'd like/be okay with it, I can put together an in-depth 'Feature' post in discussions about features I'd like to see both with the UI/UX Design and Node options once I've had more time spent with the program as it exists now. A sort of "wishlist for the future" suggestion post from a semi-pro user.
I'm not a dev, but I utilize image manipulators and other node-based workflows on a near-daily basis as a part of my freelance work, so I can certainly provide my thoughts as it pertains to my interests and use-cases.

**Reply by Correct-Syntax** [reply]
Feel free. That sounds good.
